src/Nethermind/Nethermind.Config/BlocksConfig.cs

```diff
@@ -24,6 +24,7 @@ public class BlocksConfig : IBlocksConfig
         public ulong SecondsPerSlot { get; set; } = 12;
 
         public bool PreWarmStateOnBlockProcessing { get; set; } = true;
+        public int PreWarmStateConcurrency { get; set; } = 0;
```

Comment:
This could replace `PreWarmStateOnBlockProcessing` flag if set to negative?

Reply:
Sure. That would be a possible user facing breaking change though. Is that fine?

Reply:
@rubo ?